seedlink_plotter/seedlink_plotter.py

```diff
@@ -474,9 +497,61 @@ def get_events(self):
         with self.lock:
             start = min([tr.stats.starttime for tr in self.stream])
             end = max([tr.stats.endtime for tr in self.stream])
-        neries_emsc = Client("EMSC")
-        events = neries_emsc.get_events(starttime=start, endtime=end,
-                                          minmagnitude=self.args.events)
+        
+        events = Catalog()
+        
+         # get the coordinats of the station
+        splitted_ids = [x.split(".") for x in self.args.ids]  # use seedlink id to get network, station, location and channel
+        for client in ["iris-federator","eida-routing"]:
+            try:
+                Cli = RoutingClient(client)
+                station_inventory = Cli.get_stations(network = splitted_ids[0][0],
+                                   station = splitted_ids[0][1],
+                                   channel = splitted_ids[0][3],
+                                   location = splitted_ids[0][2])
+                
+                station_latidude = station_inventory[0][0].latitude 
+                station_longitude = station_inventory[0][0].longitude 
+                break
+            except:
+               pass
+        
+        
+        # search for events fullfilling each criteria
+        for criteria in self.args.events_criteria:
+            print("Search for events with criteria" + str(criteria))
```

Comment:
space after criteria otherwise the second string is concatenated without a space.